src/pytorch_lightning/README.md

```diff
@@ -126,28 +126,28 @@ pip install pytorch-lightning['extra']
 conda install pytorch-lightning -c conda-forge
 ```
 
-#### Install stable 1.6.x
+#### Install stable 1.7.x
 
-the actual status of 1.6 \[stable\] is following:
+The actual status of 1.7 \[stable\] is the following:
```

Comment:
```suggestion
The actual status of stable release is the following:
```
how about we just say stable, else we might forget to update here

Reply:
Let's not address this here please. I don't want to go down the rabbit hole of changing the entire Readme. The intention of this PR was just to update all links with pytorch-lightning to Lightning-AI.

Reply:
The suggestion is great but it's not my area of expertise to update this.